app/models/doctor.rb

```diff
@@ -0,0 +1,5 @@
+class Doctor < ApplicationRecord
+  has_many :doctor_times
+  has_many :appointments
+  has_many :time_schedules
```

Comment:
Please ensure that the models have proper validation to prevent the storage of inaccurate data in the database.